account/applications/methods/postApplications.md

```diff
@@ -109,10 +119,17 @@ Content-Type: application/xml
         <ServiceType>Voice-V2</ServiceType>
         <AppName>Production Server</AppName>
         <CallInitiatedCallbackUrl>https://yourSecureSite.com/callbacks/voice</CallInitiatedCallbackUrl>
+        <CallInitiatedMethod>POST</CallInitiatedMethod>
+        <CallStatusCallbackUrl>https://yourSecureSite.com/callbacks/voice/status</CallStatusCallbackUrl>
+        <CallStatusMethod>POST</CallStatusMethod>
         <CallbackCreds>
             <UserId>Your-User-id</UserId>
-            <Password>Your-Password</Password>
```

Comment:
Why did this get removed?

Reply:
We do not give the password back in plaintext in our response